components/ledger/pkg/storage/sqlstorage/ledger/accounts.go

```diff
@@ -25,8 +26,8 @@ const (
 type Accounts struct {
 	bun.BaseModel `bun:"accounts,alias:accounts"`
 
-	Address  string         `bun:"address,type:varchar,unique,notnull"`
-	Metadata map[string]any `bun:"metadata,type:jsonb,default:'{}'"`
+	Address  string            `bun:"address,type:varchar,unique,notnull"`
+	Metadata map[string]string `bun:"metadata,type:jsonb,default:'{}'"`
```

Comment:
Should we plan a SQL migration for this ? Do we know what we currently have in the database?

Reply:
Yes we will need a proper migration for this.